### PR TITLE
Adding the google script and the button for onetapsignin

### PR DESCRIPTION
**Pull request by bluzeey**
Adding the google script and button for one tap sign in . Will perhaps need to check oauth-consent screen .

### CODE REVIEW

File: src/Main/NavBar.jsx [hunk 1]
```diff
@@ -4,6 +4,28 @@ import logo from "../assets/images/assembo-icon-small.png";
 import Box from '@mui/material/Box';
 import Toolbar from '@mui/material/Toolbar';
 import IconButton from '@mui/material/IconButton';
+import Button from '@mui/material/Button';
+import axios from '../axios';
+
+const google=window.google;
+
+function handleCredentialResponse(response) {
+  console.log("Encoded JWT ID token: " + response.credential);
```
Comment (yumochi) [inline src/Main/NavBar.jsx]: remove test code

File: src/Main/NavBar.jsx [hunk 2]
```diff
@@ -4,6 +4,28 @@ import logo from "../assets/images/assembo-icon-small.png";
 import Box from '@mui/material/Box';
 import Toolbar from '@mui/material/Toolbar';
 import IconButton from '@mui/material/IconButton';
+import Button from '@mui/material/Button';
+import axios from '../axios';
+
+const google=window.google;
```
Comment (yumochi) [inline src/Main/NavBar.jsx:12]: can we have some comments on this?
- explain how we are able to access window.google
- explain this is a 3rd party code, which is why we are exposing a object via the window object

File: src/Main/NavBar.jsx [hunk 3]
```diff
@@ -4,6 +4,28 @@ import logo from "../assets/images/assembo-icon-small.png";
...
+
+function handleCredentialResponse(response) {
+  console.log("Encoded JWT ID token: " + response.credential);
+  axios.post('/',{
+    token:response.credential
+  })
+}
+function oneTapSignIn(){
+    google.accounts?.id.initialize({
+          client_id: "6477792390-o5ibvubommbu11vhlglimjc0tor67reb.apps.googleusercontent.com",
```
Comment (yumochi) [inline src/Main/NavBar.jsx]: can we move this client id into an environment variable?

File: src/Main/NavBar.jsx [hunk 4]
```diff
@@ -16,7 +38,7 @@ export default function NavBar() {
           <IconButton>
             <img src={logo} className="App-logo" alt="logo" style={ { height:"50px",margin: "0px" } }/> 
           </IconButton>
-          
+          <Button id="buttonDiv" onClick={()=>oneTapSignIn()} style={{marginLeft:"85%"}}>Login</Button>
```
Comment (yumochi) [inline src/Main/NavBar.jsx]: can we most the css style into a corresponding css file

**Review by yumochi — CHANGES_REQUESTED** [review]
refer to comments

File: src/Main/NavBar.jsx (hunk 1 above)
Comment (yumochi) [inline src/Main/NavBar.jsx]: please remove testing code

File: src/Main/NavBar.jsx (hunk 3 above)
Comment (yumochi) [reply src/Main/NavBar.jsx]: Thank you

File: src/Main/NavBar.jsx [hunk 5]
```diff
@@ -12,7 +12,8 @@ import {useStyles} from './styles';
 const google=window.google;
 
 function handleCredentialResponse(response){
-  axios.get('login',{withCredentials:false,'Authorization':response.credential})
+  console.log(response.credential)
```
Comment (yumochi) [inline src/Main/NavBar.jsx]: remove testing code please

Comment (bluzeey) [reply src/Main/NavBar.jsx]: Yeah , I actually had removed test code in another commit after this.https://github.com/assembo/note-taker/pull/20/commits/4a4d65703703a6d513af3156e75c3ce3304f292c

**Review by yumochi — DISMISSED** [review]

**Review by yumochi — APPROVED** [review]
Fixed my request error

**Review by yumochi — APPROVED** [review]

**Pull request merged**